content/developer/howtos/scss_tips.rst

```diff
@@ -0,0 +1,207 @@
+=========
+SCSS Tips
+=========
+
+There are many ways to lean and simplify SCSS. The first step is to establish if custom code is needed at all.
+
+Odoo's webclient has been designed to be modular, meaning that (potentially all) classes can be shared across views.
+Check the code before creating a new class. Chances are that there is already a class or a tag doing exactly what you're looking for.
+
+
+On top of that, Odoo relies on `Bootstrap <https://getbootstrap.com/docs/5.1/getting-started/introduction/>`_ (BS), one of the most complete CSS framework available. The framework has been customized in order to match Odoo design (both community and enterprise versions), meaning that you can use any BS class, directly in Odoo and achieve a visual result consistent with our UI.
+
+.. warning::
+   The fact that a class achieve the desired visual result doesn't necessarily mean that's the right one for the job. Be aware of classes triggering JS behaviors, for example.
+
+.. warning::
+   Take care about classes semantic. To apply a **button class** to a **title** is not just semantically wrong, it may also lead to migration issues and visual inconsistencies.
+
+
+The following sections describe tips to strip-down SCSS lines **when custom-code is the only way to go**.
+
+Browsers defaults
+=================
+
+By default, each browser renders content using an *user agent stylesheet*.
+To overcome inconsistencies, some of these rules are overridden by `Bootstrap Reboot <https://getbootstrap.com/docs/5.1/content/reboot/>`_.
```

Comment:
```suggestion
To overcome inconsistencies between browsers, some of these rules are overridden by `Bootstrap Reboot <https://getbootstrap.com/docs/5.1/content/reboot/>`_.
```

Reply:
Isn't that clear enough thanks to the prev. sentence?

Reply:
I always think it's best to write as though the reader was stupid 😅
But it's not necessarily clear because the sentence can be understood in different ways.